### ISSUE TITLE
CronJob deadlink in /docs/concepts/workloads

### ISSUE THREAD

**Issue by wilsonehusin** [kubernetes/website#24676]
### Problem
In the [documentation page for `Workloads`](https://kubernetes.io/docs/concepts/workloads/), the link to `CronJob` results in 404 Page Not Found. I believe [this is the corresponding Markdown file](https://github.com/kubernetes/website/blame/45fa1517328807788d67cc633e3f8da6492e55a2/content/en/docs/concepts/workloads/_index.md#L34).

While the easy fix is to change that line to the appropriate page, I have also noticed that the `CronJob` page favors singular form (`CronJob`) over the plural form (`CronJobs`).

### Proposed Solution
I personally prefer to have consistency and follow the term understood in a manifest, i.e. `kind: CronJob`. Therefore, my first solution is:
- rename the file `cron-jobs.md` to `cron-job.md`
- fix hyperlink in `Workloads` from `cronjob` to `cron-job`
- fix everywhere else (currently working) hyperlinks from `cron-jobs` to `cron-job`

That being said, I don't know if there are implications of that much change (one thing that came to mind -- how are we maintaining consistency across translations?). Therefore, the alternative solution just to have hyperlinks work is:
- fix hyperlink in `Workloads` from `cronjob` to `cron-jobs`

**TL;DR** I don't know if we should fix everything to be right or make it so that everywhere is wrong _but_ it works :smile:

### Page(s) to update
First solution: https://github.com/kubernetes/website/search?q=cron-jobs
Second solution: https://kubernetes.io/docs/concepts/workloads/

#### Notes
/kind bug
/language en
